cms/search/views.py

```diff
@@ -63,7 +65,7 @@ def search(_class):
         search_results = Page.objects.none()
 
     # Pagination
-    paginator = Paginator(search_results, 10)
+    paginator = Paginator(search_results, RESULTS_PER_PAGE)
```

Comment:
Assuming `RESULTS_PER_PAGE` moves:

```suggestion
    paginator = Paginator(search_results, settings.RESULTS_PER_PAGE)
```

Reply:
Done